bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/rollershutter.xml

```diff
@@ -8,7 +8,7 @@
 	<!-- -->
 	<!-- Velux rollershutter Binding -->
 	<!-- -->
-	<thing-type id="rollershutter">
+	<thing-type id="rollershutter" extensible="position, limitMinimum, limitMaximum, vanePosition">
```

Comment:
This statement means that you might have multiple channels of given types. I am not sure if that's what you expect, for sure roller shutter can't have two positions at the same time.

Reply:
I suspected as much.. 

I think `extensible` is not very well specified -- it should probably have `minInstances` / `maxInstances` attributes for each of the named channel-types. Or something like that..

In this case we need..
- position => minInstances:1, maxInstances:1
- limitMinimum => minInstances:1, maxInstances:1
- limitMaximum => minInstances:1, maxInstances:1
- vanePosition => minInstances:0, maxInstances:1

Reply:
PS for this case I will implement whatever the OH maintainers instruct me to do..
- if you want an `extensible` attribute, please tell me the exact attribute value
- if you do not want an `extensible` attribute, please say so

:)

Reply:
If you want the users to be able to add channels themselves via the UI, you should have it, otherwise no.

Reply:
Ok. I just deleted it..